src/components/Form/Label.js

```diff
@@ -19,14 +20,17 @@ Label.propTypes = {
   ...propTypes.color,
   ...propTypes.typography,
 };
+
 Label.defaultProps = {
   lineHeight: 0,
   fontFamily: 'secondary',
   fontSize: 'label',
+  fontWeight: 'bold',
```

Comment:
This now accidentally bolds all children of `Label` e.g. see `TextareaMD`

probably needs a tiny bit of css on `Label`, something like:
```
& * {
  font-weight: initial;
}
```

Reply:
Somehow GitHub hid this from me. I'll update this now